src/include/pbs_ifl.h

```diff
@@ -442,6 +442,8 @@ enum mgr_obj {
 	MGR_OBJ_HOST,		/* Host  	*/
 	MGR_OBJ_HOOK,		/* Hook         */
 	MGR_OBJ_PBS_HOOK,	/* PBS Hook     */
+	MGR_OBJ_JOBARRAY_PARENT,	/* Job array parent */
```

Comment:
I am not sure whether we need design doc because all this are not visible to client, this values are only visible upto IFL layer, in batch_status (which is output and visible to client) doesn't have any member which stores this value.
So I don't think we need design document for this member, but if you and @bhroam still thinks we need design, I will create one.

Reply:
Check the docs.  If they talk about PBS_OBJ_JOB, then we should document the two new OBJ types.  JOB is weird because it isn't used directly from any IFL API call.